Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -123,6 +123,10 @@ def setup_test_formula(name, content = nil)
         def install
           (prefix/"foo"/"test").write("test") if build.with? "foo"
           prefix.install Dir["*"]
+          (buildpath/"test.c").write \
+            "#include <stdio.h>\\nint main(){printf(\\"test\\");}"
```

Comment:
I was so hoping for the first nested heredoc. :wink:

But in all seriousness, you might want to add a `return 0;` to that `main` function, as some older versions of Clang tend to produce code that generates a random exit code otherwise. (Which will be annoying if we decide to use this formula/binary to test `brew test`.)